### DISCUSSION TITLE
Issues with deployment to production, using Github Pages and Netlify

### DISCUSSION THREAD

**Discussion by SeeYangZhi** [vuepress/core#599] [Q&A] [answered]
I have currently tried using the default vuepress-next and vuepress-webpack^2.0.0-beta.32 package to deploy to production. But both attempts produced HTML pages without styles on Netlify and just a normal markdown doc using Github page. 

Is anyone encountering a similar issue?

Edit: Was able to solve the issue on GitHub pages by having the correct source under the GitHub Pages settings.

**Comment by Mister-Hope** [comment, marked as answer]
It did be an issue last year. This issue should be fixed now, as I am deploying my theme docs and demo on netlify:

- https://vuepress-theme-hope-v2.netlify.app/
- https://vuepress-theme-hope-v2-demo.netlify.app/

If you are still having issues, welcome to provide a minial reproduction repo link in a new issue.
